doc/architecture/decisions/0005-rooms-and-beds-for-both-cas-services.md

```diff
@@ -0,0 +1,106 @@
+                 |           |PremiseCharacteristic      |
+                 |           |---------------------------|
+                 |           |*premise_id: integer       |
+                 |           |*characteristic_id: integer|
+                 |           `---------------------------'
+                 |                           .|.
+                 |                            |
+                 |                            |
+                .|.                  ,---------------------------.
+      ,---------------------------.  |Premise                    |
+      |RoomCharacteristic         |  |---------------------------|
+      |---------------------------|  |*id: integer <<autonumber>>|
+      |*room_id: integer          |  |--                         |
+      |*characteristic_id: integer|  |address: string            |
+      `---------------------------'  |postcode: string           |
+             .|.                     `---------------------------'
+              |-------`                      |
+                     |             `----------
+                     |             .|.
+               ,---------------------------.
+               |Room                       |
+               |---------------------------|
+               |*id: integer <<autonumber>>|
+               |--                         |
+               `---------------------------'
+                              |
+                             .|.
+               ,---------------------------.
+               |Bed                        |
+               |---------------------------|
+               |*id: integer <<autonumber>>|
+               |--                         |
+               `---------------------------'
+
+[2] [PlantUML](https://www.plantuml.com/plantuml/uml/fP5VQiCm3CRVVGhJnu8l44eftGN6Bb0yMsl5OYj8og0qlVkIT0dDC2omn-Zxau-V3wAUjKsraK_CZMUVR8qPzDg09TOB6GDPKQ_svTAhGO5H58Ez6Mkt62kBMU4CR7SxtohbDhq3x_UzwjnWoJv1PKKPytdu5k0l372IGCqizE487xwjjRARxMQCRnD5gF_otQTvRqIfD97x--bfvt0B2zz2dVjU6H8Ab2gRX_IFrhNqV8mC8a_y6X8D56SYIm3hUkV84UBQXFa5ekkSerk1UjbOncbCuTNkv5pNZSrcJfKSFwAPNEElkiUciNWw726Q0-JOfleR)
+
+## Consequences
+
+- For CAS3, the new 'create a Room' endpoint will also need to create a bed.
+  The implementation for this will need to be considered to allow CAS1 to do
+  this across two different transactions
+- Splitting out characteristics into a new table, potentially with join tables,
+  will require more set up work but this cost should be reasonable
+- Admin may in future need to modify the list of characteristics, having these
+  stored in a table rather than hardcoded should enable that
+- CAS1 will have to maintain their temporary `Premises.bed_count` mechanism for
+  beds alongside this extension work until such time as it can be refactored
+- We may learn of a Characteristic in future that isn't a boolean that may
+  require us to revist this plan to accommodate multiple values
```

Comment:
Would it be helpful to document the fact that we neither service currently anticipates the need to model the characteristics of an individual bed?

Reply:
Yes that was something we noticed wasn't it 👍